fbw-a32nx/src/wasm/systems/a320_systems/src/hydraulic/sfcc/flaps_channel.rs

```diff
@@ -95,6 +115,21 @@ impl FlapsChannel {
         }
     }
 
+    fn get_cas(
+        &self,
+        adirs: &impl AdirsMeasurementOutputs,
+    ) -> (Option<Velocity>, Option<Velocity>) {
+        let cas1 = adirs.computed_airspeed(1).normal_value();
+        let cas2 = adirs.computed_airspeed(2).normal_value();
+
+        match (cas1, cas2) {
+            (Some(cas1), Some(cas2)) => (Some(cas1), Some(cas2)),
+            (Some(cas1), None) => (Some(cas1), Some(cas1)),
+            (None, Some(cas2)) => (Some(cas2), Some(cas2)),
+            (None, None) => (None, None),
+        }
```

Comment:
This can be simplified to:
```suggestion
        (cas1.or(cas2), cas2.or(cas1))
```